src/Core/Domain/Common/Contracts/AuditableEntity.cs

```diff
@@ -1,6 +1,6 @@
 namespace DN.WebApi.Domain.Common.Contracts;
 
-public abstract class AuditableEntity : BaseEntity, IAuditableEntity, ISoftDelete
+public abstract class AuditableEntity<T> : BaseEntityWith<T>, IAuditableEntity, ISoftDelete
```

Comment:
Same here, if you add 

```
public abstract class AuditableEntity : AuditableEntity<Guid>
{}
```
The default is guid and you don't have to change all the other places.

Reply:
good point